perf-baseline/reporter/README.md

```diff
@@ -0,0 +1,9 @@
+# Reporter
```

Comment:
I recommend linking to this directory from the "Related configs" section of the top-level README, similar to how you're also linking to the `grafana` and `linkerd1-perf` subdirs.